scss/_base_reset.scss

```diff
@@ -1,408 +1,473 @@
+
+  /**
+   * 1. Remove the gray background on active links in IE 10.
+   * 2. Remove gaps in links underline in iOS 8+ and Safari 8+.
+   */
+
   a {
-    background-color: transparent;
-    -webkit-text-decoration-skip: objects;
+    background-color: transparent; /* 1 */
+    -webkit-text-decoration-skip: objects; /* 2 */
+  }
+
+  /**
+   * Remove the outline on focused links when they are also active or hovered
+   * in all browsers (opinionated).
+   */
+
+  a:active,
+  a:hover {
+    outline-width: 0;
   }
 
-  // Remove the bottom border in Firefox 39- & Add the correct text decoration
-  // in Chrome, Edge, IE, Opera, and Safari.
+  /**
+   * 1. Remove the bottom border in Firefox 39-.
+   * 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
+   */
+
   abbr[title] {
-    border-bottom: 0;
-    text-decoration: underline;
+    border-bottom: none; /* 1 */
+    text-decoration: underline; /* 2 */
+    text-decoration: underline dotted; /* 2 */
   }
 
-  blockquote,
-  q {
-    quotes: none;
+  /**
+   * Prevent the duplicate application of `bolder` by the next rule in Safari 6.
+   */
+
+  b,
+  strong {
+    font-weight: inherit;
   }
 
-  legend {
-    border: 0;
+  /**
+   * Add the correct font weight in Chrome, Edge, and Safari.
+   */
+
+  b,
+  strong {
+    font-weight: bolder;
   }
 
-  figure {
-    margin: 0;
+  /**
+   * 1. Correct the inheritance and scaling of font size in all browsers.
+   * 2. Correct the odd `em` font sizing in all browsers.
+   */
+
+  code,
+  kbd,
+  samp {
+    font-family: monospace, monospace; /* 1 */
+    font-size: 1em; /* 2 */
+  }
+
+  /**
+   * Add the correct font style in Android 4.3-.
+   */
+
+  dfn {
+    font-style: italic;
+  }
+
+  /**
+   * Add the correct background and color in IE 9-.
+   */
+
+  mark {
+    background-color: #ff0;
+    color: #000;
+  }
+
+  /**
+   * Add the correct font size in all browsers.
+   */
+
+  small {
+    font-size: 80%;
   }
 
-  abbr,
-  acronym {
-    cursor: help;
+  /**
+   * Prevent `sub` and `sup` elements from affecting the line height in
+   * all browsers.
+   */
+
+  sub,
+  sup {
+    font-size: 75%;
+    line-height: 0;
+    position: relative;
+    vertical-align: baseline;
   }
 
-  .link-arrow:after {
-    content: '\0000a0›';
+  sub {
+    bottom: -0.25em;
   }
 
-  nav ul li h2 a:after {
-    content: '\0000a0›';
+  sup {
+    top: -0.5em;
   }
 
-  nav ul li a:after,
-  ul li p a:after {
-    content: '';
+  /* Embedded content
+     ========================================================================== */
+
+  /**
+   * Add the correct display in IE 9-.
+   */
+
+  audio,
+  video {
+    display: inline-block;
   }
 
+  /**
+   * Add the correct display in iOS 4-7.
+   */
+
+  audio:not([controls]) {
+    display: none;
+    height: 0;
+  }
+
+  /**
+   * Remove the border on images inside links in IE 10-.
+   */
+
   img {
-    border: 0;
-    height: auto;
-    max-width: 100%;
```

Comment:
We need to retain this styling outside of the reset as its relied on heavy to hold together layouts.